src/Components/Pages/TransferPage.tsx

```diff
@@ -533,7 +533,7 @@ const TransferPage = () => {
       <div className={classes.wrapper}>
         <div className={classes.header}>
           {process.env.REACT_APP_MAINTENANCE === "false" &&
-            (!isReady ? (
+            (!isReady && !transactionStatus ? (
```

Comment:
Could you assign the whole expression to a variable with a good name?

Reply:
Fixed